DASCharter-2021.html

```diff
@@ -532,7 +532,7 @@ <h3 id="tentative">
           Tentative Deliverables
         </h3>
         <p>
-          Depending on the progress, the Group may also produce W3C Recommendations for the following documents:
+          Depending on the progress, including interest from multiple implementers, the Group may also produce W3C Recommendations for the following documents:
```

Comment:
```suggestion
          Depending on the progress, including consideration for adequate implementation experience, the Group may also produce W3C Recommendations for the following documents:
```

I'd suggest we refer to the W3C Process term "[adequate implementation experience](https://www.w3.org/2020/Process-20200915/#adequate-implementation)" to remove ambiguity around what constitutes "interest".